constellations/gemini-user/src/announcements/announce.sass

```diff
@@ -0,0 +1,20 @@
+@import "../utils/_constants.sass"
+
+#view-announce
+  h1
+    width: 100%
+    margin: 90px 0px 50px
```

Comment:
Use $spacing from _constants.sass for spacing purposes. In design, we often want things to be "unit-proportional" meaning spaces are multiples of a unit (i.e. 8px). So a large space could be 32px or 64px because they are multiples of 8.